### ISSUE TITLE
Content for a welcome message

### ISSUE THREAD

**Issue by jdotrjs** [phaser-discord/administration#1]
Our welcome channel will be the first thing people should see when they join the server. It needs to bake into it the rules, how we operate, when we're active, some good getting started guides, how how to ask for help, etcetcetc.

Some initial thoughts:

- [Devlog #116](https://madmimi.com/p/a022cb) has some links to v3 tutorials; we should cherry pick the best and the order newbies should approach them
- [Wild's v3? video](https://www.youtube.com/watch?v=St_tPsG0pX0) should be there for folks wondering (maybe ping him to update it if applicable)
- Some good v2 intros
- describe @helper or whatever approach we come up with to allow for people to indicate that they're around and have time to assist new folks

**Comment by 16patsle** [comment]
Here's what I have for now:

```markdown
**Welcome to the server!**
Please take the time to read through this message.

**Channel overview**
#general – For general discussion about Phaser and making games
#beginners – For questions about the basics, or other beginner questions
#phaser-3 – For discussing Phaser 3-specific concepts, APIs and code
#discord-feedback – Ideas and feedback for the server

**Rules**
Be nice.
(More to come)

**Important people**
- photonstorm (aka Rich) , the guy behind Phaser. Comes online here from time to time

- @16patsle#7801 (that's me), moderator and admin
- @jdotr#9225, moderator and admin
- @Arcan#8543, moderator and helper person
```

**Comment by jdotrjs** [comment]
Nice, since this is live I'm going to mark it as completed. We can make updates as other things get resolved (wiki, rules for the space, etc)

**Comment by 16patsle** [comment]
Did a quick update, since I forgot to add something about getting started guides and such:
```markdown
**Welcome to the server!**
Please take the time to read through this message.

**Channel overview**
#general – For general discussion about Phaser and making games
#beginners – For questions about the basics, or other beginner questions
#phaser-3 – For discussing Phaser 3-specific concepts, APIs and code
#discord-feedback – Ideas and feedback for the server

**Rules**
Be nice.
http://phaser.io/community/be-nice
(More to come later)

**Guidelines**
If you have a question about Phaser, feel free to mention @Ping for help.

**Getting started with Phaser**
We'll soon make a list of some good beginner tutorials for Phaser.
In the mean time, this official tutorial is a good start (both v3 and v2/CE)
http://phaser.io/tutorials/making-your-first-phaser-game

Wondering if you should use Phaser 3 or Phaser 2/CE? @WiLD#4832 has a nice video on that:
https://www.youtube.com/watch?v=St_tPsG0pX0
In short, v3 has the fanciest new features, but v2/CE has better docs, tutorials, and community content.

**Important people**
- photonstorm (aka Rich) , the guy behind Phaser. Comes online here from time to time

- @16patsle#7801 (that's me), moderator and admin
- @jdotr#9225, moderator and admin
- @Arcan#8543, moderator and helper person
```
